starter-core/src/test/groovy/io/micronaut/starter/feature/camunda/Platform7Spec.groovy

```diff
@@ -83,8 +85,28 @@ class Platform7Spec extends ApplicationContextSpec implements CommandOutputFixtu
       <artifactId>h2</artifactId>
       <scope>runtime</scope>
     </dependency>
+''') == 1
+        template.count('''\
```

Comment:
Please, use the new BuildVerifierApi to test dependencies.

Reply:
Thanks for the hint.
I switched to BuildTestVerifier (I guess that is what you are referring to)